aws/resource_aws_codebuild_project_test.go

```diff
@@ -1399,6 +1419,46 @@ resource "aws_codebuild_project" "test" {
 `, oName, rName)
 }
 
+func testAccAWSCodeBuildProjectConfig_Environment_RegistryCredential(rName string) string {
+	return testAccAWSCodeBuildProjectConfig_Base_ServiceRole(rName) + fmt.Sprintf(`
+resource "aws_codebuild_project" "test" {
+  name         = %[1]q
+  service_role = "${aws_iam_role.test.arn}"
+
+  artifacts {
+    type = "NO_ARTIFACTS"
+  }
+
+  environment {
+    compute_type                = "BUILD_GENERAL1_SMALL"
+    image                       = "2"
+		type                        = "LINUX_CONTAINER"
```

Comment:
Can you please fix the formatting here? Looks like spaces versus tabs. 😄 

```suggestion
  type                        = "LINUX_CONTAINER"
```